test/Main.hs

```diff
@@ -0,0 +1,208 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Move these to the `default-extensions` of the `test-suite` in the `.cabal` file.